docs/whats-new/dotnet-docs-2021-08-01.md

```diff
@@ -0,0 +1,122 @@
+- [Profiling Interfaces](../framework/unmanaged-api/profiling/profiling-interfaces.md) - Add orphans to TOC
+- [Resgen.exe (Resource File Generator)](../framework/tools/resgen-exe-resource-file-generator.md) - .NET DI-ready localization content
+
+## .NET fundamentals
+
+### New articles
+
+- [File globbing in .NET](../core/extensions/file-globbing.md)
+- [OutputType not changed from Exe to WinExe for Windows Forms and WPF projects](../core/compatibility/sdk/6.0/outputtype-not-set-automatically.md)
+- [CA2018: The `count` argument to `Buffer.BlockCopy` should specify the number of bytes to copy](../fundamentals/code-analysis/quality-rules/ca2018.md)
+- [dotnet workload restore](../core/tools/dotnet-workload-restore.md)
+- [Blazor server: Disable long polling fallback transport](../core/compatibility/aspnet-core/6.0/blazor-long-polling-fallback.md)
+- [JsonNode no longer supports C# `dynamic` type](../core/compatibility/serialization/6.0/jsonnode-dynamic-type.md)
+- [Localization in .NET](../core/extensions/localization.md)
+- [Perform culture-insensitive case changes](../core/extensions/performing-culture-insensitive-case-changes.md)
+- [Perform culture-insensitive string comparisons](../core/extensions/performing-culture-insensitive-string-comparisons.md)
+- [Retrieve resources in .NET apps](../core/extensions/retrieve-resources.md)
+- [FrameworkReference replaced with WindowsSdkPackageVersion for Windows SDK](../core/compatibility/sdk/5.0/override-windows-sdk-package-version.md)
+- [How to use source generation in System.Text.Json](../standard/serialization/system-text-json-source-generation.md)
+- [HTTP with .NET](../core/extensions/http-client.md)
+- [C# code in templates not supported by earlier versions](../core/compatibility/sdk/6.0/csharp-template-code.md)
+- [System.Drawing.Common only supported on Windows](../core/compatibility/core-libraries/6.0/system-drawing-common-windows-only.md)
+- [IL2026: Members attributed with RequiresUnreferencedCode may break when trimming](../core/deploying/trim-warnings/il2026.md)
+- [Caching in .NET](../core/extensions/caching.md)
+- [dotnet format](../core/tools/dotnet-format.md)
+
+### Updated articles
+
+- [.NET Generic Host](../core/extensions/generic-host.md) - Added host shutdown details
+
+## Azure SDK for .NET
+
+### New articles
+
+- [Resource management using the Azure SDK for .NET](../azure/sdk/resource-management.md)
+
+### Updated articles
+
+- [Resource management using the Azure SDK for .NET](../azure/sdk/resource-management.md)
+  - docs(azure .net sdk): update resource management intro
+  - Add .NET guide for Track 2 Management SDK
+
+## C# language
+
+### New articles
+
+- [Selection statements (C# reference)](../csharp/language-reference/statements/selection-statements.md)
+
+### Updated articles
+
+- [using directive](../csharp/language-reference/keywords/using-directive.md) - C# 10 - Add articles for global using
```

Comment:
We should add the file scoped namespaces, even though it's a bit smaller as a change:

```suggestion
- [using directive](../csharp/language-reference/keywords/using-directive.md) - C# 10 - Add articles for global using
- [namespace keyword](../csharp/language-reference/keywords/namespace.md) - C# 10 - Add file scoped namespace
```